src/main/java/Check.java

```diff
@@ -0,0 +1,72 @@
+public class Check {
+     //Function to check if an input has mark as the first word
+     public static boolean isMark(String input){
+        if(input.length()>=4){
+            if("mark".equalsIgnoreCase(input.substring(0,4))){
```

Comment:
Would be nice if there is a space between the bracket and if as such:
```suggestion
            if ("mark".equalsIgnoreCase(input.substring(0,4))) {
```